modules/nodes-scheduler-pod-topology-spread-constraints-configuring.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-scheduler-pod-topology-spread-constraints
+
+[id="nodes-scheduler-pod-topology-spread-constraints-configuring_{context}"]
+= Configuring pod topology spread constraints
+
+The following steps demonstrate how to configure pod topology spread constraints to distribute pods that match the specified labels based on their zone.
+
+You can specify multiple pod topology spread constraints, but you must ensure that they do not conflict with each other. All pod topology spread constraints must be satisfied for a pod to be placed.
+
+.Prerequisites
+
+* A cluster administrator has added the required labels to nodes.
+
+.Procedure
+
+. Create a `Pod` spec and specify a pod topology spread constraint:
++
+.Example `pod-spec.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: my-pod
+  labels:
+    foo: bar <1>
```

Comment:
We do not have comments for this part , please remove it. thank.

Reply:
Good catch, thanks!